### ISSUE TITLE
"On Page Load" is triggered twice after Authentication/Registration

### ISSUE THREAD

**Issue by nicmohr25** [FlutterFlow/flutterflow-issues#1289]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

If you're using the authentication action to register or log in as a user, the "On page load" action on the home page, to which this auth action navigates you, is triggered twice.

This happens when using:
- Phone Authentication
- E-Mail Authentication
- Apple Authentication
- Gmail Authentication

Due to this behaviour API Calls are made twice, Actions are triggered twice which resolves in Bottom Sheets and Message Boxes beeing shown twice.

This Error is also described in #790 but nobody is answering this post for a few weeks now.

### Expected Behavior

Navigation after Authentication triggers "On Page Load" only once.

### Steps to Reproduce

1. Create a LogIn/Register Page
2. Create a Auth Home Page which displays a bottom Sheet in the Action "On Page Load"
3. Run the App and authenticate
4. You are navigated to the Auth Home Page
5. Bottom Sheet is displayed twice

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFTlcmB34hNodcF+KrqdcJslWQsCUMdXOczitVEew4bCIj4OrMuPPTBP1VtZ+KzaH1YAlX+kTgE0M7Gi/PxCcJdHEisRrpEwZFIYBPPbDmVMozTEs2eS3dDL99bfRW60reFmxJSCLBeWE5h2Dn3N+eiTT+Cf9qOYwx5e6fDbOY=

### Context

Showing Information on page load after registration line warnings, infos or a informations collection form

### Visual documentation

-

### Additional Info

-

### Environment

```markdown
- FlutterFlow version: v3.1 - Flutter 3.10.4
- Platform: Web
- Browser name and version: Chrome Version 115.0.5790.170
- Operating system and version affected: macOS Ventura Version 13.4.1
```

**Comment by hariprasadms** [comment]
Hi @nicmohr25 - Thanks for posting the issue.  Sorry for looking at this issue bit delayed. I can confirm that, I can reproduce the issue.

**Comment by marinosabijan** [comment]
Are you sure the API's are triggered twice from the on page load actions? I think the issue is that there are two dialogs/bottom sheets that pop up because of flutter state context issues of how FF is handling it for the on page load actions after authenticating. To fix this, I add 2 actions to close down the dialogs/bottom sheets.

**Comment by alexsanderp** [comment]
@marinosabijan Yes, the APIs are called twice. Which page load action is performed twice.

**Comment by nicmohr25** [comment]
Yes, I am positive. API Calls made in the on page load action are triggered twice. i just checked again because it has been some time. 

![on_page_load_issue](https://github.com/FlutterFlow/flutterflow-issues/assets/120254395/e7641983-d864-4496-b527-145ae9bc0b30)

And your workaround with two "close bottom sheet/dismiss dialog" actions seems to work. I also tried this back then but I can remember that the page crashed when using the "close bottom sheet" action twice when there is no bottom sheet left to close. But I couldn't reproduce this error at the moment. 

But it is still no clean solution.

**Comment by aemelyanovff** [comment]
Duplicate of https://github.com/FlutterFlow/flutterflow-issues/issues/1289

**Comment by alexsanderp** [comment]
@aemelyanovff Did you close this issue because it was a duplicate of #1289 and did you close #1289 because it was a duplicate of this one? I didn't understand.

**Comment by alexsanderp** [comment]
@aemelyanovff Did you close the issue because it was a duplicate of itself? I didn't understand.

**Comment by nicmohr25** [comment]
Closed because it’s a duplicate of itself? The other one was #790 as mentioned. 790 was already closed and there where no answers for WEEKS

so kindly open the Ticket again. 

@aemelyanovff 
@hariprasadms

**Comment by alexsanderp** [comment]
So it's a duplicate of #790, okay...

You mentioned in this issue that it was a duplicate of 1290, but it is 1290. Now I understand.

**Comment by aemelyanovff** [comment]
Sorry I meant it's a duplicate of https://github.com/FlutterFlow/flutterflow-issues/issues/790
